docs/ensureSymlink.md

```diff
@@ -1,30 +1,32 @@
-# ensureSymlink(srcpath, dstpath[, type][, callback])
+# ensureSymlink(srcPath, destPath[, type][, callback])
 
 Ensures that the symlink exists. If the directory structure does not exist, it is created.
 
 **Alias:** `createSymlink()`
 
-- `srcpath` `<String>`
-- `dstpath` `<String>`
+- `srcPath` `<String>`
+- `destPath` `<String>`
 - `type` `<String>`
+  - It is only available on Windows and ignored on other platforms. It can be set to `dir`, `file`, or `junction`.
```

Comment:
```suggestion
- `type` `<String>` It is only available on Windows and ignored on other platforms. It can be set to `dir`, `file`, or `junction`.
```